pilot/proxy/envoy/watcher.go

```diff
@@ -206,11 +206,12 @@ type envoy struct {
 func NewProxy(config meshconfig.ProxyConfig, node string) proxy.Proxy {
 	// inject tracing flag for higher levels
 	var args []string
-	if glog.V(4) {
+	if log.DebugEnabled() {
 		args = append(args, "-l", "trace")
-	} else if glog.V(3) {
+	} /* TODO(nmittler): When should the "debug" flab be used?
```

Comment:
@kyessenov  any thoughts on how to handle this?

Reply:
This will require a new flag.

Reply:
@kyessenov I've created #2257 to address this.